test/addons/layout-change/lib/test-cuddlefish-loader.js

```diff
@@ -21,11 +21,6 @@ exports["test compatibility"] = function(assert) {
   assert.equal(require("tabs"),
                require("sdk/tabs"), "sdk/tabs -> tabs");
 
-  if (app.is("Firefox")) {
-    assert.equal(require("widget"),
-                 require("sdk/widget"), "sdk/widget -> widget");
-  }
```

Comment:
I'm not sure why we should test for a module that it doesn't exist anymore, I'm missing the point to understand what's the advantage to do so. It seems to me that we're still adding a reference to widget module that soon or later we have to remove again.